### DISCUSSION TITLE
Notification when repository is deleted

### DISCUSSION THREAD

**Discussion by ochalet-wp** [community/community#21826] [New to GitHub]
<p>Hi,</p>
<p>I would like to know if there’s a way in Github to receive a notification when a given repository is deleted?</p>
<p>Cheers</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>I don’t believe GitHub will send you a notification for that currently. But, you could set up something with the GitHub API to make sure the repo still exists. </p>

**Comment by GrahamCampbell** [comment]
<p>Actually, you can setup a webhook to fire when the repo is deleted.</p>

**Comment by logankilpatrick** [comment]
<p>Yes! The only constraint is that you would need to ask them to set that up for you if you aren’t an org/repo owner. In my mind, it seemed like the use case was for a repo that the person was not associated with or part of. </p>
